dapr/clients/grpc/client.py

```diff
@@ -451,6 +455,56 @@ def save_state(
         return DaprResponse(
             headers=call.initial_metadata())
 
+    def execute_transaction(
+            self,
+            store_name: str,
+            operations: Sequence[TransactionalStateOperation],
+            metadata: Optional[MetadataTuple] = ()) -> DaprResponse:
+        """Saves or deletes key-value pairs to a statestore as a transaction
+
+        This saves or deletes key-values to the statestore as part of a single transaction,
+        metadata can be passed with metadata field.
+
+        The example saves states to a statestore:
+            from dapr import DaprClient
+            with DaprClient() as d:
+                resp = d.execute_transaction(
+                    store_name='state_store',
+                    operations=[
+                        TransactionalStateOperation(key=key, data=value),
```

Comment:
I think operation type is missing in this example.

Reply:
It defaults to 'upsert'.